pkg/ssh/ssh.go

```diff
@@ -68,7 +68,9 @@ func init() {
 
 // TODO: Unit tests for this code, we can spin up a test SSH server with instructions here:
 // https://godoc.org/golang.org/x/crypto/ssh#ServerConn
-type SSHTunnel struct {
+
+// TheTunnel struct define the ssh info.
+type TheTunnel struct {
```

Comment:
I dislike the "The"Tunnel. Can we come up with something a bit better for the name? SecureTunnel maybe? Honestly I preferred SSHTunnel but the linter won't accept it.

Reply:
I thought about this, but not sure how to name more appropriate? Can be modified according to your suggestion, use the “SecureTunnel”. 😃

Reply:
All the change has done, please check and review again, thank you!